bson_extract.go

```diff
@@ -108,6 +108,7 @@ func extractMetricsFromValue(val *birch.Value) (extractedMetrics, error) {
 	case bsontype.DateTime:
 		metrics.values = append(metrics.values, birch.VC.Int64(epochMs(val.Time())))
 		metrics.types = append(metrics.types, bsontype.DateTime)
+		metrics.ts = val.Time()
```

Comment:
Do we know if the _id will be a DateTime or a Timestamp?

Reply:
The _id field will be a DateTime field. This is recorded in the ftdc [poc](https://github.com/10gen/support-tools/tree/master/ftdc#data-capture-container-format
) and [spec](https://docs.google.com/document/d/1CRUnSIs8C-giQ-eyT1CGcuViFVxiKHjPpku1z1HWyEk/edit#heading=h.6w1ntcru8hdl).